big_tests/tests/acc_e2e_SUITE_data/acc_test_helper.erl

```diff
@@ -53,3 +72,13 @@ alter_message({From, To, Acc, Packet}) ->
     PCh2 = lists:keyreplace(<<"body">>, 2, PCh, NewBody),
     {From, To, Acc, {xmlel, PName, PAttrs, PCh2}}.
 
+cached_my_jid(User, Acc) ->
```

Comment:
I don't like this function being noisy in the logs, even on success.
I would also consider splitting it into two functions or even eliminating it completely and moving parts of the code to the callers.